### PR TITLE
Refactor: refine the prescription list and prescription add pages

### PR DESCRIPTION
**Pull request by stu01509**
#### Short description of what this resolves:
Refine the prescription list and prescription add pages

Remove outdated code and style.
Add the Bootstrap grid system, form, and table style.
Adjust the color and icon of the buttons to follow the styling guide.

image

image

### CODE REVIEW

**Comment by sjpadgett** [comment]
Thanks for using btn-sm where possible. Especially in dialogs.
I can't think of any good reason to use btn-lg anywhere!!!

**Comment by stu01509** [comment]
Yeah, use btn-lg looks like very odd lol

**Comment by bradymiller** [comment]
hi @stu01509 , Code looks great. Will wait for @tywrenn 's blessing and then will bring it into codebase.

File: templates/prescription/general_list.html [hunk 1]
```diff
@@ -161,183 +161,181 @@
...
+                                        {else}
+                                        <!-- TajEmo work by CB 2012/06/14 02:16:32 PM target="_script" opens better -->
+                                            <a target="_script" id="multiprintcss" href="{$CONTROLLER}prescription&multiprintcss&id={$printm|attr_url}" onclick="top.restoreSession()" class="btn btn-primary btn-print">{xlt t='View Printable Version'} ({xlt t='HTML'})</a>
+                                        {/if}
+                                        </td>
+                                        {if $GLOBALS.rx_use_fax_template}
+                                        <td style="border-style:none;">
+                                            <a id="multiprintToFax" href="{$CONTROLLER}prescription&multiprintfax&id={$printm|attr_url}" onclick="top.restoreSession()" class="btn btn-primary btn-download">{xlt t='Download'} ({xlt t='Fax'})</a>
+                                        </td>
```
Comment (stu01509) [inline templates/prescription/general_list.html:195]: Hi @tywrenn 

I think this `style="border-style:none;"` maybe can remove or change to `border-0` ?

Comment (tywrenn) [reply templates/prescription/general_list.html:195]: @stu01509 Yeah change it to `border-0`

**Comment by stu01509** [comment]
@tywrenn  Updated the commit.

File: templates/prescription/general_list.html [hunk 2]
```diff
@@ -161,183 +161,181 @@
...
-    <table class="w-100">
-        <tr>
-            <td align="left">
-                <table>
-                    <tr>
-                        <td>
-                        {if $GLOBALS.rx_zend_pdf_template}
-                            <a target="_blank" id="multiprint" href="{$GLOBALS.webroot}/{$GLOBALS.baseModDir}{$GLOBALS.zendModDir}/public/prescription-pdf-template/{$GLOBALS.rx_zend_pdf_action}?id={$printm|attr_url}" onclick="top.restoreSession()" class="btn btn-primary">{xlt t='Download'} ({xlt t='PDF'})</a>
+        {if $prescriptions}
+        <h3>{xlt t='List'}</h3>
```
Comment (tywrenn) [inline templates/prescription/general_list.html:168]: This file needs some grid and form-group improvements

File: templates/prescription/general_list.html [hunk 3]
```diff
@@ -161,183 +161,161 @@
...
-                    </td>
-                </tr>
-                </table>
-            </td>
-        </tr>
-    </table>
-</div>
-
+<body id="prescription_list">
+    <div class="container">
```
Comment (tywrenn) [inline templates/prescription/general_list.html:165]: Change this to `container-fluid`

Comment (stu01509) [reply templates/prescription/general_list.html]: Ok, updated the commit.

**Comment by tywrenn** [comment]
@bradymiller

**Comment by bradymiller** [comment]
hi @stu01509 , bringing this in. thanks for yet another great improvement :) -brady

**Pull request merged**